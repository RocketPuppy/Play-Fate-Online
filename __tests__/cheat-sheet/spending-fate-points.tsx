import * as React from "react";
import * as renderer from "react-test-renderer";
import SpendingFatePoints from "../../src/cheat-sheet/spending-fate-points";

test("renders all the pieces", () => {
  const render = renderer.create(<SpendingFatePoints />);

  expect(render.toJSON()).toMatchSnapshot();
});
