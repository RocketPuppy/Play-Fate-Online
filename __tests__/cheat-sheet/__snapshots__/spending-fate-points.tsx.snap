// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Spending Fate Points
  </h2>
  <p>
    Spend fate points to
  </p>
  <ul>
    <li>
      Invoke an aspect
    </li>
    <li>
      Power a stunt
    </li>
    <li>
      Refuse a compel
    </li>
    <li>
      Declare a story detail
    </li>
  </ul>
</div>
`;
